semantic/tasks/watch.js

```diff
@@ -0,0 +1,231 @@
+/*******************************
+           Watch Task
+*******************************/
+
+var
+  gulp         = require('gulp-help')(require('gulp')),
+
+  // node dependencies
+  console      = require('better-console'),
+  fs           = require('fs'),
+
+  // gulp dependencies
+  autoprefixer = require('gulp-autoprefixer'),
+  chmod        = require('gulp-chmod'),
+  clone        = require('gulp-clone'),
+  gulpif       = require('gulp-if'),
+  less         = require('gulp-less'),
+  minifyCSS    = require('gulp-clean-css'),
+  plumber      = require('gulp-plumber'),
+  print        = require('gulp-print').default,
+  rename       = require('gulp-rename'),
+  replace      = require('gulp-replace'),
+  uglify       = require('gulp-uglify'),
+  replaceExt   = require('replace-ext'),
+  watch        = require('gulp-watch'),
+
+  // user config
+  config       = require('./config/user'),
+
+  // task config
+  tasks        = require('./config/tasks'),
+  install      = require('./config/project/install'),
+
+  // shorthand
+  globs        = config.globs,
+  assets       = config.paths.assets,
+  output       = config.paths.output,
+  source       = config.paths.source,
+
+  banner       = tasks.banner,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['banner' is assigned a value but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182945)